src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsCard.component.ts

```diff
@@ -110,20 +124,30 @@ export class NotebookViewsCardComponent implements OnInit {
 		}
 	}
 
+	public set executionState(state: CellExecutionState) {
+		if (this._executionState !== state) {
```

Comment:
Why the check? There's no other logic to skip so doesn't seem necesssary.

(and why not just make the property itself public?)